src/it/includes-excludes/src/test/java/org/apache/maven/it0055/PersonTest.java

```diff
@@ -18,14 +18,10 @@
  */
 package org.apache.maven.it0055;
 
-import junit.framework.TestCase;
-
```

Comment:
We could replace JUnit 3 dependency by JUnit 5, then replace above line by:

```java
import static org.junit.jupiter.api.Assertions.assertEquals;
```

which would allow us to keep the `assertEquals` in `PersonTest`. Here too, the real test is only to ensure that the compiler found the `assertEquals` method.